queue_services/entity-bn/src/entity_bn/bn_processors/registration.py

```diff
@@ -241,11 +242,40 @@ def _get_program_account(identifier, transaction_id):
         return None, str(err)
 
 
-def _get_firm_legal_name(business: Business):
-    """Get firm legal name."""
-    parties = [party_role.party for party_role in business.party_roles.all()
-               if party_role.role.lower() in (PartyRole.RoleTypes.PARTNER.value,
-                                              PartyRole.RoleTypes.PROPRIETOR.value)]
+def _sort_party_name(party):
+    """Sort party's name """
+    organization_name = party.get('organization_name', '')
+    last_name = party.get('last_name', '')
+    first_name = party.get('first_name', '')
+    middle_initial = party.get('middle_initial', '')
+
+    sort_name = (
+        organization_name +
+        (' ' + last_name if last_name else '') +
+        (' ' + first_name if first_name else '') +
+        (' ' + middle_initial if middle_initial else '')
+    )
+    return sort_name.strip()
```

Comment:
is this used somewhere?

Reply:
nope.... only for testing, forgot to delete